tests/unit/disco/sagas/testDiscovery.js

```diff
@@ -0,0 +1,104 @@
+import SagaTester from 'redux-saga-tester';
```

Comment:
sure, I can call it `testDisco` if you want. I like disco. https://www.youtube.com/watch?v=yuLz4T6YXWs